public/ACMW/css/events.css

```diff
@@ -40,7 +31,7 @@
 .navbar {
 	font-family: Montserrat, sans-serif;
 	margin-bottom: 0;
-	background-color: #2d2d30;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

Reply:
here also..

Reply:
Have a look at this: [Disallow !important](https://github.com/CSSLint/csslint/wiki/Disallow-!important)